### ISSUE TITLE
Time to interactable content as comparison value

### ISSUE THREAD

**Issue by StephanHoyer** [koglerjs/threaditjs#3]
... would be nice

**Comment by koglerjs** [comment]
Agreed, especially if we can get info on mobile devices.  There's a lot of potential for comparative metrics that I'm looking into; if you've got more ideas I'm definitely taking notes.

**Comment by lhorie** [comment]
Some other ideas:

- byte size (bonus points for size after minifcation+gzip)
- library caveats: e.g. whether there are hard dependencies, whether a library was required because the base framework didn't have that functionality (e.g. for routing, ajax), whether there are ecosystem libraries to fill the gaps (e.g. angular-router, vue-router), and whether libraries are typically not used in production (e.g. jsx transformer)

**Comment by uxnow** [comment]
:+1:
